GOVERNANCE.md

```diff
@@ -0,0 +1,179 @@
+maintainer members.
+
+# Support
+
+All participants in the community are encouraged to provide support for new
+users within the project management infrastructure. This support is provided as
+a way of growing the community. Those seeking support should recognise that all
+support activity within the project is voluntary and is therefore provided as
+and when time allows. A user requiring guaranteed response times or results
+should therefore seek to purchase a support contract from a community member.
+However, for those willing to engage with the project on its own terms, and
+willing to help support other users, the community support channels are ideal.
+
+# Contribution process
+
+Anyone can contribute to the project, regardless of their skills, as there are
+many ways to contribute. For instance, a contributor might be active on the
+project mailing list and issue tracker, or might supply patches.
+
+# Decision making process
+
+Decisions about the future of the project are made through discussion with all
+members of the community, from the newest user to the most experienced
+maintainer member. All non-sensitive project management discussion takes place
+on the project contributors’ mailing list. Occasionally, sensitive discussion
+occurs on a private list.
+
+In order to ensure that the project is not bogged down by endless discussion
+and continual voting, the project operates a policy of lazy consensus. This
+allows the majority of decisions to be made without resorting to a formal vote.
+
+## Lazy consensus
+
+Decision making typically involves the following steps:
+
+- Proposal
+- Discussion
+- Vote (if consensus is not reached through discussion)
+- Decision
+
+Any community member can make a proposal for consideration by the community. In
+order to initiate a discussion about a new idea, they should send an email to
+the project contributors’ list or submit a patch implementing the idea to the
+issue tracker (or version-control system if they have commit access). This will
+prompt a review and, if necessary, a discussion of the idea. The goal of this
+review and discussion is to gain approval for the contribution. Since most
+people in the project community have a shared vision, there is often little
+need for discussion in order to reach consensus.
+
+In general, as long as nobody explicitly opposes a proposal or patch, it is
+recognised as having the support of the community. This is called lazy
+consensus - that is, those who have not stated their opinion explicitly have
+implicitly agreed to the implementation of the proposal.
+
+Lazy consensus is a very important concept within the project. It is this
+process that allows a large group of people to efficiently reach consensus, as
+someone with no objections to a proposal need not spend time stating their
+position, and others need not spend time reading such mails.
+
+For lazy consensus to be effective, it is necessary to allow at least 72 hours
```

Comment:
IMHO it is not reasonable to expect objections to everything within 72 hours. We have PRs without feedback open for this amount of time already and it does not seem like everyone agrees with them directly.

Reply:
What is your suggestion?

Reply:
In order to allow smooth progress, I think it is reasonable to allow 72h or even less for another opinion to be heard. If there is no opinion at all, then it is a stale PR which has nothing to do with a consensus. But if there is already an opinion and no one is heard challenging it, we can assume there is no objection and take it in.
We already do this at much shorter periods, and I think it is good.